site/mex_invenio/assets/semantic-ui/js/mex_invenio/includes/DisplayValue.js

```diff
@@ -41,10 +41,11 @@ const DisplayValues = ({
           </div>
 
           <Button
+            type="button"
             className="link-like"
             onClick={() => setShowAll(!showAll)}
           >
-            {showAll ? 'Show less ▲' : `Show all (${total}) ▼`}
+            {showAll ? `_("Show less") ▲` : `_("Show all") (${total}) ▼`}
```

Comment:
We need to use the `t` function here. Have a look at the README (https://github.com/robert-koch-institut/mex-invenio/blob/feature/develop/site/mex_invenio/README.md) and I use it in the Record component (https://github.com/robert-koch-institut/mex-invenio/blob/feature/develop/site/mex_invenio/assets/semantic-ui/js/mex_invenio/Record.js)